py/scripts/gimli_hash.py

```diff
@@ -0,0 +1,15 @@
+# Part of liblithium, under the Apache License v2.0.
```

Comment:
Consider adding `#!/usr/bin/env python3` to stand-alone scripts.

Reply:
They were there when this was a standalone script, but this is using the `setuptools` `entry_points` mechanism to produce the scripts, and this file itself is not a runnable script (and also does not have execute permissions). See @xialu000's comment here: https://github.com/teslamotors/liblithium/pull/46#discussion_r1024702117

Reply:
I've renamed the directory to avoid confusion.